lib/news_scraper/transformers/article.rb

```diff
@@ -20,22 +19,26 @@ def transform
         transformed_response.merge(root_domain: root_domain)
       end
 
+      def self.scrape_patterns
```

Comment:
Also, the trainer and external classes all reference the class variable, not the cached scrape_patterns on article